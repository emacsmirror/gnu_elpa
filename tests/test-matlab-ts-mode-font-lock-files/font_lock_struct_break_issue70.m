% -*- matlab-ts -*-

% See https://github.com/acristoffers/tree-sitter-matlab/issues/70

AllowableMenus = struct(...
    'edit', 'on', ...
    'delete', 'on', ...
    'break', 'on');

AllowableMenus.break
