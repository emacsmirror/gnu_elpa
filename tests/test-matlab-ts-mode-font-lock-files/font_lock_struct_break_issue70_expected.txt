c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

vvvvvvvvvvvvvv o BBBBBBbccc
    SssssSD SssSD ccc
    SssssssSD SssSD ccc
    SsssssSD SssSbD

ddddddddddddddDddddd
